documentation/src/main/asciidoc/topics/data_container/ref_eviction_strategies.adoc

```diff
@@ -0,0 +1,27 @@
+[id='eviction_strategies_{context}']
+[[eviction_strategies]]
+= Eviction Strategies
+
+`NONE`::
+Eviction is not enabled and it is assumed that the user will not invoke evict directly
+on the cache. If passivation is enabled this will cause aa warning message to be
+emitted. This is the default strategy.
+
+`MANUAL`::
+This strategy is just like <b>NONE</b> except that it asssumes the user will be
```

Comment:
To be more minimalist, you should say something like "Same as **NONE**, but the user must evict the cache directly." It's a pet peeve of mine, but evict is not a noun.